src/main/java/org/imsglobal/pox/IMSPOXRequest.java

```diff
@@ -548,6 +548,8 @@ public static void sendReplaceResult(String url, String key, String secret, Stri
 		HttpPost request = buildReplaceResult(url, key, secret, sourcedid, score, resultData, isUrl);
 		DefaultHttpClient client = new DefaultHttpClient();
 		HttpResponse response = client.execute(request);
+System.out.println("**** outcomes status code: "+response.getStatusLine().getStatusCode());
```

Comment:
Are these really intended to be permanently in the library?  Wouldn't a logging library be better then using "System.out.println" in a library call that other systems would be using?